examples/REST_API.md

```diff
@@ -0,0 +1,26 @@
+The following example demonstrates how to launch a workflow via REST API. To find out more about available API calls please refer to "[Check Digdag REST API](https://github.com/treasure-data/digdag)".
+
+1. Launch a server with
+```
+$ digdag server --memory
+```
+2. Create and push the following workflow to the server (observe the value of id)
+```
+$ mkdir resttest
+$ cd resttest
+$ cat << 'EODAG' > resttest.dig
+timezone: Europe/Zurich
+
++echoparam:
+  echo>: ${msg}
+EODAG
+
+$ digdag push resttest
+```
+3. Call the workflow via REST API with curl to pass a message (workflowId has to be the value from the previous command)
+```
+$ curl -X PUT "http://localhost:65432/api/attempts" \
+       -H  "accept: application/json" \
+       -H  "Content-Type: application/json" \
+       -d "{ \"params\": { \"msg\": \"Hello from REST API.\" }, \"sessionTime\": \"$(date -Iseconds)\", \"workflowId\": \"1\"}"
+```
```

Comment:
Does this example work on your platform?
I got the following error. (CentOS 7)

```
curl -X PUT "http://localhost:65432/api/attempts" \
>        -H  "accept: application/json" \
>        -H  "Content-Type: application/json" \
>        -d "{ \"params\": { \"msg\": \"Hello from REST API.\" }, \"sessionTime\": \"$(date -Iseconds)\", \"workflowId\": \"1\"}"
{"message":"Invalid ISO time format: %s2019-05-01T18:24:57+0900 (through reference chain: io.digdag.client.api.ImmutableRestSessionAttemptRequest$Json[\"sessionTime\"])","status":400}
```

Digdag allows iso8601 format like `2019-05-01T13:38:52+09:00` only. It require colon like '+09:00'. However `date -Iseconds` command returns `+0900`(without the colon).

What do you think to use date string like `2019-05-01T13:38:52+09:00` explicitly until fix format issue?

Reply:
@hiroyuki-sato: this is interesting. There are slightly different outputs on other platforms:

FreeBSD:
```
$ uname -srvm
FreeBSD 12.0-RELEASE-p3 FreeBSD 12.0-RELEASE-p3 GENERIC  amd64
$ date -Iseconds
2019-05-01T21:12:41+02:00
```

Linux (RHEL7):
```
$ uname -srvm
Linux 3.10.0-957.10.1.el7.x86_64 #1 SMP Thu Feb 7 07:12:53 UTC 2019 x86_64
$ date -Iseconds
2019-05-01T21:13:31+0200
```

The following expression would produce the desired output in both cases:
```
$ date -Iseconds | sed -e 's|\([0-9]\{2\}\)\([0-9]\{2\}\)$|\1:\2|'
```

On a side note, I was wondering if it might be desirable to be able to specify a keyword such as `now` which would interpreted internally as the current moment?

Please let me know if the proposed workaround is acceptable. I would then update the pull request.